lang/da.js

```diff
@@ -0,0 +1,21 @@
+/* eslint quotes: 0 */
+
+export default {
+	"appliedFilters": "Anvendte filtre:",
+	"noActiveFilters": "Ingen aktive filtre",
+	"filterRemoved": "Filteret {filterText} er fjernet",
+	"clearFilters": "Ryd filtre",
+	"allFiltersRemoved": "Alle filtre er fjernet",
+	"categoryTitleMultiple": "{title} ({numSelected})",
+	"searchBy": "Søg efter {category}",
+	"clear": "Ryd",
+	"filter": "Filtrér",
+	"filterBy": "Filtrér efter",
+	"filterMultiple": "Filtrér: {numOptions} filtre",
+	"filterSingle": "Filtrér: 1 filter",
+	"more": "flere",
+	"searchPagedResults": "{rangeStart}-{rangeEnd} af {totalCount} resultater",
+	"searchPagedResultsForQuery": "{rangeStart}-{rangeEnd} af {totalCount} resultater for \"{query}\"",
+	"searchResults": "{totalCount} resultater",
+	"searchResultsForQuery": "{totalCount} resultater for \"{query}\""
```

Comment:
Should this and `de.js` have `sort` and `sortWithOption` with English terms?

Reply:
Good catch -- added!